test/functional/host/errata/list_test.rb

```diff
@@ -0,0 +1,48 @@
+require File.join(File.dirname(__FILE__), '../../test_helper')
+require_relative '../../lifecycle_environment/lifecycle_environment_helpers'
+
+describe 'host errata listing' do
+  include LifecycleEnvironmentHelpers
+
+  before do
+    @cmd = %w(host errata list)
+  end
+
+  let(:org_id) { 1 }
+  let(:host_id) { 2 }
+  let(:lifecycle_env_id) { 3 }
+  let(:empty_response) do
+    {
+      "total" => 0,
+      "subtotal" => 0,
+      "page" => "1",
+      "per_page" => "1000",
+      "error" => nil,
+      "search" => nil,
+      "sort" => {
+        "by" => nil,
+        "order" => nil
+      },
+      "results" => []
+    }
+  end
+
+  it "lists the host errata belonging to a lifecycle-environment by name" do
+    params = ["--host-id=#{host_id}", "--organization-id=#{org_id}", '--lifecycle-environment=test']
+
+    expect_lifecycle_environment_search(org_id.to_s, 'test', lifecycle_env_id)
+
+    ex = api_expects(:host_errata, :index, 'host errata list') do |par|
+      par['host_id'] == host_id && par['environment_id'] == lifecycle_env_id &&
+        par['page'] == 1 && par['per_page'] == 1000
+    end
```

Comment:
You'll get better failure output if you use `.with_params(...)` like [this](https://github.com/Katello/hammer-cli-katello/blob/49b48b3c1d431da10c254e3c59e62552cb909122/test/functional/content_view/filter/create_test.rb#L29).

Reply:
so the downside of with params is that you have to specify all key values. If we are only testing a subset, we are better off testing this way.

Reply:
True. Which params aren't we checking for?

Reply:
That being said I decided to comply with the comment :)